packages/webpack/src/plugin.js

```diff
@@ -483,6 +483,15 @@ class LavaMoatPlugin {
         // I stole the idea from Zach of module federation fame
 
         const onceForChunkSet = new WeakSet()
+
+        if (typeof options.scuttleGlobalThis === 'object') {
+          if (typeof options.scuttleGlobalThis.exceptions === 'object') {
+            options.scuttleGlobalThis.exceptions.forEach((exception, i, exceptions) => {
+              exceptions[i] = exception.toString()
+            })
+          }
+        }
```

Comment:
https://github.com/LavaMoat/LavaMoat/blob/74acc1ca62e0a5792faf137c7623b90ff8c41e8d/packages/webpack/src/plugin.js#L660

typeof `exceptions` is always `string[] | undefined` right?
(so currently not reaching inside the condition)

Reply:
could maybe tidy the nested `if` too, i'll add suggestion

Reply:
```suggestion
        if (
          typeof options.scuttleGlobalThis === 'object' &&
          typeof options.scuttleGlobalThis.exceptions
        ) {
          options.scuttleGlobalThis.exceptions =
            options.scuttleGlobalThis.exceptions.map((exception) =>
              exception.toString()
            )
        }
```

Reply:
(the new var names lgtm)